packages/grafana-ui/src/components/IconButton/IconButton.tsx

```diff
@@ -23,12 +23,12 @@ export interface Props extends React.ButtonHTMLAttributes<HTMLButtonElement> {
   /** Type of the icon - mono or default */
   iconType?: IconType;
   /** Tooltip content to display on hover */
-  tooltip?: PopoverContent;
+  tooltip: PopoverContent;
```

Comment:
agreed it's probably a breaking change... fortunately it's a very easy fix for any plugin developer. i'm not sure what we can do about it other than mark this PR as `breaking change` and add a message explaining how to fix it (add some `tooltip` text)? 🤔

Reply:
I added the label `breaking change`.

> add some tooltip text

@ashharrison90 Do you mean adding a default text in case there is anything set? Something like:

``` javascript
export const IconButton = React.forwardRef<HTMLButtonElement, Props>(
  (
    {
      name,
      size = 'md',
      iconType,
      tooltip = 'Add tooltip here',
      tooltipPlacement,
      ariaLabel,
      className,
      variant = 'secondary',
      ...restProps
    },
    ref
  ) => {}

```
Isn't this too aggressive?

Reply:
@L-M-K-B please take a look at https://github.com/grafana/grafana/blob/main/contribute/breaking-changes-guide.md#i-know-its-a-breaking-change-whats-next 

And please reach out to @grafana/plugins-platform-frontend to see if we want to merge this breaking change or rather not.

Reply:
@tolzhabayev Thanks for pointing this out. From the documentation about breaking changes I got that it might take quite a while until a breaking change will be released. That's why I decided to remove the breaking change from this PR and merge it separately (see [new PR](https://github.com/grafana/grafana/pull/69699)). So we can get the direct a11y improvements of this PR out a lot faster.